providers/shared/inputseeders/shared/id.ftl

```diff
@@ -247,6 +253,23 @@
     ]
 [/#function]
 
+[#-- Define and setup the layer data so that it can be used by the plugin transformer --]
+[#function shared_configseeder_layers filter state]
```

Comment:
Are we missing a configseeder or does this just need a name change?

```
{
[2021-03-24T23:36:49.186Z]   "Timestamp": "2021-03-24T23:36:48.597Z",
[2021-03-24T23:36:49.186Z]   "Severity": "fatal",
[2021-03-24T23:36:49.186Z]   "Message": "Unable to invoke any of the input step function options for Config pipeline - check function naming",
[2021-03-24T23:36:49.186Z]   "Context": {
[2021-03-24T23:36:49.186Z]     "Id": "shared",
[2021-03-24T23:36:49.186Z]     "Priority": 100,
[2021-03-24T23:36:49.186Z]     "Sources": [
[2021-03-24T23:36:49.186Z]       "*"
[2021-03-24T23:36:49.186Z]     ],
[2021-03-24T23:36:49.186Z]     "Function": "configseeder"
[2021-03-24T23:36:49.186Z]   },
[2021-03-24T23:36:49.186Z]   "Detail": [
[2021-03-24T23:36:49.186Z]     [
[2021-03-24T23:36:49.186Z]       "shared",
[2021-03-24T23:36:49.187Z]       "configseeder",
[2021-03-24T23:36:49.187Z]       "fixture",
[2021-03-24T23:36:49.187Z]       "mock"
[2021-03-24T23:36:49.187Z]     ],
[2021-03-24T23:36:49.187Z]     [
[2021-03-24T23:36:49.187Z]       "shared",
[2021-03-24T23:36:49.187Z]       "configseeder",
[2021-03-24T23:36:49.187Z]       "fixture"
[2021-03-24T23:36:49.187Z]     ],
[2021-03-24T23:36:49.187Z]     [
[2021-03-24T23:36:49.187Z]       "shared",
[2021-03-24T23:36:49.187Z]       "configseeder"
[2021-03-24T23:36:49.187Z]     ]
[2021-03-24T23:36:49.187Z]   ]
[2021-03-24T23:36:49.187Z] }
```

Reply:
I think that was from #1597 I've rebased now so it should be fine